spec/run_oc_pedant.rb

```diff
@@ -43,7 +43,15 @@
     '--skip-omnibus',
     '--skip-usags',
     '--exclude-internal-orgs',
-    '--skip-headers'
+    '--skip-headers',
+
+    # Chef 12 features not yet 100% supported by Chef Zero
```

Comment:
Hi @tyler-ball, I've filed https://github.com/chef/chef-zero/issues/153 to track this.